suites/test_search_beta.py

```diff
@@ -55,9 +56,40 @@ def search_for_repo(self):
         utilities.click_element(self.driver, By.CLASS_NAME, locators.CANCEL_BUTTON_ON_REPO_SEARCH_BAR_CLASS_NAME)
         utilities.wait(1)
 
+    @lcc.test("Verify that user is able to filter results using find by name, verify listed columns and pagination")
+    def find_by_name(self):
+        search_beta_page.select_repo(self.driver, self.repo_name)
+        utilities.wait(10)
+        utilities.enter_text(self.driver, By.CSS_SELECTOR, locators.SEARCH_TITLE_CSS, constants.published_module)
+        utilities.click_element(self.driver, By.CSS_SELECTOR, locators.TITLE_SEARCH_ICON_CSS)
+        utilities.wait(10)
+        check_that("entered title can be seen in search results", utilities.get_text(self.driver,
+                    By.CSS_SELECTOR, locators.FIRST_MODULE_LISTED_CSS), contains_string(constants.published_module))
+        check_that("Title is displayed on search page", utilities.find_element(self.driver, By.XPATH,
+                           locators.TITLES_ON_SEARCH_PAGE_XPATH).is_displayed(), is_true())
+        check_that("Repository is displayed on search page", utilities.find_element(self.driver, By.XPATH,
+                           locators.REPOSITORY_ON_SEARCH_PAGE_XPATH).is_displayed(), is_true())
+        check_that("Upload date is displayed on search page", utilities.find_element(self.driver, By.XPATH,
+                           locators.UPLOAD_DATE_ON_SEARCH_PAGE_XPATH).is_displayed(), is_true())
+        check_that("Last Published Date is displayed on search page", utilities.find_element(self.driver, By.XPATH,
+                           locators.LAST_PUBLISHED_DATE_ON_SEARCH_PAGE_XPATH).is_displayed(), is_true())
+        check_that("pagination is enabled on search page", utilities.find_element(self.driver, By.XPATH,
```

Comment:
This test will surely fail as we are filtering the titles which returns < 5 titles hence, pagination will always be disabled

Reply:
changes has been pushed .. thanks for your review :)